gadgets/bpfstats/README.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: bpfstats
+sidebar_position: 0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# bpfstats
+
+The bpfstats Gadget provides CPU and memory usage for Gadgets and eBPF programs.
+
+## Getting started
```

Comment:
Please add the "Exporting metrics" section (like [this](https://inspektor-gadget.io/docs/latest/gadgets/profile_blockio#exporting-metrics)) describing the metrics this gadget is able to export with one simple example.